src/Nfts/Nfts.tsx

```diff
@@ -1,73 +1,97 @@
 import {useNavigation} from '@react-navigation/native'
 import React from 'react'
-import {Image, RefreshControl, ScrollView, StyleSheet, Text, View, ViewProps} from 'react-native'
+import {defineMessages, useIntl} from 'react-intl'
+import {Image, RefreshControl, ScrollView, StyleSheet, Text, View} from 'react-native'
+import {SafeAreaView} from 'react-native-safe-area-context'
 
 import noNftsImage from '../assets/img/no-nft.png'
-import {OfflineBanner, Spacer, StatusBar} from '../components'
+import {Icon, OfflineBanner, Spacer, StatusBar} from '../components'
 import {useNfts} from '../hooks'
 import {WalletStackRouteNavigation} from '../navigation'
 import {useSelectedWallet} from '../SelectedWallet'
 import {ImageGallery, SkeletonGallery} from './ImageGallery'
 
-export const Nfts = () => {
+type Props = {
+  search?: string
+}
+
+export const Nfts = ({search}: Props) => {
+  const strings = useStrings()
   const wallet = useSelectedWallet()
-  const {nfts, isLoading, refetch, isRefetching, error} = useNfts(wallet)
+  const {nfts, isLoading, refetch, isRefetching, isError} = useNfts(wallet, {search})
   const navigation = useNavigation<WalletStackRouteNavigation>()
 
   const showDetails = (id: string) => navigation.navigate('nft-details-routes', {screen: 'nft-details', params: {id}})
   const handleNFTSelect = (index: number) => showDetails(nfts[index].id)
 
-  if (error !== null) {
-    return (
-      <View style={styles.root}>
-        <StatusBar type="dark" />
-        <View style={styles.container}>
-          <OfflineBanner />
-          <Text>Error loading NFTs</Text>
-        </View>
-      </View>
-    )
-  }
-
   return (
     <View style={styles.root}>
       <StatusBar type="dark" />
```

Comment:
is it necessary?

Reply:
I believe it's being used on every screen to configure the status bar color explicitly, but I could be wrong on this one